### ISSUE TITLE
twin import event

### ISSUE THREAD

**Issue by b0ykoe** [tritonmc/Triton#137]
**Is your feature request related to a problem? Please describe.**
Importing the TWIN command (/twin hash) does not seem to refresh some parts of the game. We're covering that part with the PlayerLanguageChangeEvent but we're missing a event for the TWIN command.

**Describe the solution you'd like**
Implementing a TwinLanguageUpdateEvent would be a good idea for that.

**Describe alternatives you've considered**
Well we could start a scheduler and check if somethings got changed but that would be heavy on ressources.

**Additional context**

**Comment by diogotcorreia** [comment]
As an alternative for now:

On Spigot, use can use the [`PlayerCommandPreprocessEvent`](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/player/PlayerCommandPreprocessEvent.html) to listen for the `/twin` command, and if it has more than one argument (that is, it is downloading something), start a task with a delay of ~1-2s (to allow it to download and update everything) and then do whatever you want to do 😄 

However, this would only work if you're not using BungeeCord since I don't think there is a similar event there 😢

**Comment by b0ykoe** [comment]
Yeah true, its not the most important part right now but if you find the time I would appriciate it if you add the event.
